README.md

```diff
@@ -315,6 +315,10 @@ For complex/professional level work, you have optional full control of the train
 
 ```python
 class LitAutoEncoder(pl.LightningModule):
+    def __init__(self):
+        super().__init__()
+        self.automatic_optimization = False
+
```

Comment:
Can you use a `@property instead`? and lets put it below training_step

Reply:
it's already a property in LightningModule.

Reply:
Ah it's a getter/setter in the LightningModule... I thought it was just a simple getter like the one I wrote.

Not sure what's the advantage of the setter since it's a simple flag

Reply:
I see two advantages:
1) can attach docs with a docstring
2) can implement design by contract if needed, and responsibility of updating the state lies in the hands of the LightningModule not the external modifier.

2) granted, is not so relevant here in this trivial setter, but I still see value